jinja/carnival/prize_booth.html

```diff
@@ -0,0 +1,79 @@
+{% extends 'base.html' %}
+{% block content %}
+{% block navbar %}
+{% include "navbar.html" %}
+{% endblock %}
+
+<div class="parallax" style="background-image: url('../img/xanadu-background.png'); box-shadow: none;">
+    <div class="container">
+
+        <div class="mt-5 pt-5 text-center teal-text text-lighten-2">
+            <img src="../img/carnival_logo_splash.png" class="logo"></img>
+        </div>
+        <div class="row-sm text-center justify-content-center" style="color:grey; font-family: 'Roboto', sans-serif !important;">
+            <h3 class="col-12 mt-5 mb-5 grey-text">
+                Cash your points to win our amazing <strong>Carnival Prizes</strong>. 
+            </h3>
+            
+	<div style="font-size: 14pt;">
+            <p>
+            The participants with the most points will be the winners of some cool Xanadu swag and more!

```

Comment:
Please tone down the exclamation marks. It's ok to have some, but not on every sentence, it's too much